Header header

string  transcription
float64 confidence


